src/main/java/slimeknights/tconstruct/tables/client/inventory/module/SideButtonsWidget.java

```diff
@@ -12,33 +12,28 @@
 
 public class SideButtonsWidget<T extends Button> implements Widget, GuiEventListener {
 
-  protected final MultiModuleScreen<?> parent;
+  public static final int SPACING = 4;
 
-  // left or right of the parent
-  protected final boolean right;
+  protected final MultiModuleScreen<?> parent;
 
-  public int leftPos;
-  public int topPos;
-  public int imageWidth;
-  public int imageHeight;
-  public int xOffset;
-  public int yOffset;
+  public final int leftPos;
+  public final int topPos;
+  public final int imageWidth;
+  public final int imageHeight;
```

Comment:
the note about making things private applied to these too. I do not like public non-static fields on things unless needed (which they rarely are)

Reply:
Sure. I left them public as they were being used for rendering the buttons beam in `TinkerStationScreen`, which was reworked in #4988 by f62c1f03e0a4d56041f36f8923f34a81e43e8cc0. I'll make these fields private with getters for those fields that were being used.